@xen-orchestra/web-core/lib/locales/fr.json

```
@@ -77,7 +77,9 @@
   "confirm-cancel": "Êtes-vous sûr de vouloir annuler ?",
   "confirm-delete": "Vous êtes sur le point de supprimer {0}",
   "connected": "Connecté",
-  "connections": "Connexions",
+  "connecting": "Connexion en cours",
+  "connection": "Connexions",
+  "connection-failed": "Échec de la connexion.",
```

I don't think we need to add the punctuation here:
```suggestion
  "connection-failed": "Échec de la connexion",
```